Formula/blahtexml.rb

```diff
@@ -36,6 +36,7 @@ def install
       system "make", "blahtexml-mac"
     end
     on_linux do
+      ENV.deparallelize
```

Comment:
Can we add a comment with the error this fixes explaining that it is caused by a race condition?

Reply:
Updated.